e2e-tests/charge-cycles-on-reject.sh

```diff
@@ -47,7 +47,7 @@ check_charging()
 
   # Send invalid transaction.
   set +e
-  SEND_TX_OUTPUT=$(dfx canister  --wallet="${WALLET}" call --with-cycles "${EXPECTED_FEE}" bitcoin "${METHOD}" "${RECORD}" 2>&1);
```

Comment:
With dfx version update the order of keywords matter, param `--wallet` can't go before `call`.